src/frontend/apps/desk/src/features/footer/Footer.tsx

```diff
@@ -138,13 +140,20 @@ export const Footer = () => {
             </StyledLink>
           ))}
         </Box>
+
         <Text
           as="p"
           $size="m"
           $margin={{ top: 'big' }}
           $variation="600"
           $display="inline"
         >
+          {config?.RELEASE && (
+            <>
+              <Trans>Version</Trans>: {config?.RELEASE} •&nbsp;
```

Comment:
I have the feeling it gives more context like that, wdyt ?
```suggestion
              {t(`Version: {{release}}`, { release: config?.RELEASE})} •&nbsp;
```